sofar/io.py

```diff
@@ -212,11 +212,11 @@ def _read_netcdf(filename, verify, verbose, mode):
     if verify:
         try:
             sofa.verify(mode="read")
-        except: # noqa (No error handling - just improved verbosity)
+        except ValueError as e:
```

Comment:
I usally do a bare except only I experienced multiple possible errors during debugging. Passing tests must not necessarily mean that this catches all possible errors. Maybe revert to be on the safe side and provide verbose feedback?